src/app/js/views/connection.js

```diff
@@ -2,23 +2,15 @@ import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 import Peer from './peer'
 
-export
-default class Connection extends Component {
-  static displayName = 'Connection';
-  static propTypes = {
-    ipfs: PropTypes.object,
-    location: PropTypes.object,
-    bytesRead: PropTypes.number,
-    bytesWritten: PropTypes.number,
-    peer: PropTypes.object,
-    addr: PropTypes.object
-  };
-
-  state = {
-    open: false
-  };
+class Connection extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      open: false
+    }
+  }
 
-  _handleClick = () => {
+  _handleClick () {
```

Comment:
Ok, confirmed. I'll bring back this ES7 feature and added standard to use babel-eslint so that it doesn't warn on it.